src/i18n.ts

```diff
@@ -93,6 +94,13 @@ export function fromNow(date: Date): string {
   })
 }
 
+export function distanceToNowStrict(date: Date, addSuffix = false): string {
```

Comment:
considering there is already a `formatRelative`, is this new function really needed?

Reply:
There is a difference: `formatRelative`/`fromNow` shows the given date in words relative to now, e.g. "today at 4:30 AM" or "Sunday at 4:30 AM," where `distanceToNow[Strict]` basically displays the _interval_ between now and the given date, e.g. "in 12 hours" or "in 10 minutes". I chose `distanceToNow` as is what's used on desktop and it takes less screen space.

Let me know if you would prefer `formatRelative` though and I'll change it, it's an easy switch 👍

Reply:
👍